### ISSUE TITLE
qcheck does NOT work correctly

### ISSUE THREAD

**Issue by KOBA789** [rosylilly/QueenCheck#2]
see this code.

``` Ruby
class Foo
  def initalize()
    @x = 0
  end

  attr_accessor :x
end

describe Foo do
  foo = Foo.new
  qcheck(foo, :x=, [Integer]) do | result, args, err |
    foo.x.should == args[0]
  end
end
```

Generally thinking, it seems to work correctly. But, it does not work.

**Comment by rosylilly** [comment]
fixed in v1.0.0. thanks.

**Comment by rosylilly** [comment]
this api is deprecated.

please use RSpec::Core::ExampleGroup#QueenCheck

thx
